ibm-mq/1.6/modules/ROOT/pages/ibm-mq-xml-ref.adoc

```diff
@@ -549,7 +549,11 @@ Allows a user to perform a session recover when the AckMode#MANUAL mode is elect
 
 ** CORRELATION_ID
 ** MESSAGE_ID
-** NONE |  |  +++CORRELATION_ID+++ |
+** NONE | Indicates how requests and reply messages are correlated, which ensures that the incoming message is the proper reply to the outgoing message.
+
+* The `CORRELATION_ID` pattern value is used with the *Publish Consume* operation. After the operation publishes a request message to the destination queue, it expects a message with the same correlation ID as the request message. It then consumes that correlated response message.
+
+* The `MESSAGE_ID` pattern value is used with the *Publish Consume* operation. After the operation publishes a request message to the destination queue, it expects a message with a correlation ID of the same value as the message ID of the request message. It then consumes that correlated response message. | CORRELATION_ID |
```

Comment:
This text is a part of the On New Message trigger, but it is related to the publish-consume operation. There is another entry for the Request-Reply Pattern within that operation (line 331). I believe that the explanation should include another section within that operation to provide a comprehensive understanding.

Reply:
And I think it would also be useful to mention the link to the [Listen For a Reply](https://docs.mulesoft.com/ibm-mq-connector/1.6/ibm-mq-publish-consume) documentation for more information.